### ISSUE TITLE
The app cannot be found on Play store

### ISSUE THREAD

**Issue by tamtom** [ZacSweers/CatchUp#146]
do I have to do something to make ot available?  
when i click on the Google play link it redirect me to item not found page

**Comment by ZacSweers** [comment]
Yes it's temporarily taken down due to play store policy. Fixed on master
but I have other changes on master that are not ready for prime time yet.

**Comment by billpapadimas** [comment]
Can we at least get an APK of the last release or do I have to build from source? Thanks!

**Comment by ZacSweers** [comment]
You can build from source

**Comment by heyakyra** [comment]
If someone builds from source, could you please share on https://www.aptoide.com/ ?

It would also be nice to have an official build on F-Droid #180

**Comment by ZacSweers** [comment]
Please do not encourage others to put this on other app stores or use this
issue to advertise.

**Comment by heyakyra** [comment]
I was just suggesting that people share an _open source_ (if I'm not mistaken…) app with others through a convenient backup repository since not everyone will build from source. 

Anyways, what are the issues you referred to that need to be fixed before pushing back to the play store?

**Comment by rolandog** [comment]
It's such a shame that such a cool app was taken down. I really like it, and I'm really thankful for all the effort you put into it!

If I understood correctly the [twitter thread](https://mobile.twitter.com/ZacSweers/status/1098237389254860805?p=v), some dependency triggered Play Store to think you were using a tracker and, without a privacy policy in place, that's against the app store policies?

Is there an external, up-to-date Apk that you could vouch for @ZacSweers ? (perhaps signed by you?)

Or, could you list which dependencies and steps are needed to build (I'm familiar with Python, and use Makefiles for several projects, but haven't built an Android package).

After perusing the repository, I can tell that some dependencies required (to build in Ubuntu) are:

* android-sdk
* Gradle
* Kotlin (can I use the [command line compiler](https://kotlinlang.org/docs/tutorials/command-line.html)?
* npm
    * apollo-codegen (version 0.19.1)

And a lot of configuring, from what I can read in the wiki.

Anything else I'm missing?

**Comment by ZacSweers** [comment]
You can build from source locally. I still work on this but to be honest I don't feel much of a drive to maintain publishing it on the play store since it's a hobby project.

**Comment by Omkaragrawal** [comment]
Man, I tried and made a copy of this as a PWA, I had previously asked you that can we collaborate on that but you denied the request, so I dropped it, such a sham it was taken down. I am extremely sorry I cannot help building android, but if you are still interested in a PWA then do tell me. BTW I still use this app, and is one of the app with most on screen time on my mobile. Just loved the brilliant idea and implementation.
